common/thorhelper/thorcommon.hpp

```diff
@@ -323,6 +325,32 @@ class SimpleActivityTimer
         }
     }
 };
+
+class BlockedActivityTimer
```

Comment:
If TIME_ACTIVITIES is not defined this will no compile - there needs to be a dummy implementation in the #else block.